src/components/SlateEditor/EditorFooter.tsx

```diff
@@ -99,6 +105,7 @@ const EditorFooter: React.FC<Props> = ({
       showSaved={!formIsDirty && (savedToServer || isNewlyCreated)}
       onClick={onSaveClick}
       hideSecondaryButton={hideSecondaryButton}
+      disabled={formikHasErrors}
```

Comment:
Går altså an å lagre f.eks. concepts (alle de typene som ikke bruker validation i backend) selv om formiken inneholder error, så lurte på om det burde legges inn en fix på det? Kan evt bruke isValid og bare sende den inn for de typene som ikke bruker validation i backend.